frontend/cypress/integration/all.spec.js

```diff
@@ -93,6 +110,11 @@ context('All tests', () => {
       cy.get('.ant-alert-success').should('contain', 'Template submitted')
       cy.get('button').contains('Go Back').click()
     })
+    it('visit process detail page', () => {
+       navigateToMenuItem('Protocols')
+       cy.get('.anticon-eye').first().parent('a').click()
+       cy.get('body').should('contain', `Process`)
+    })
```

Comment:
Sidenote, we are we using a eye icon for more details on protocols, but the rest of the pages show the details when clicking on the name/id of the item? The UX isn't consistent.

Reply:
This is the solution we found because in that table (in Protocols) we have 3 different models being displayed, those of Protocols, Process, and ProcessSample, while we only want to represent those to the users as 'Processes'. The eye convey additional information that is accurately representated by an item id. It could have been a magnifying glass, an eye, a text with 'More details' or so. We are aware this is not the perfect solution, but it is what we have for now. Any suggestions?

Reply:
Use eyes everywhere? I don't care either way, but it should be consistent, and at the very least not at opposite sides of the screen for different tables.

Reply:
Noted. This will be handled and discussed in a separate issue.